definitions/procedures/repositories/index_katello_repositories_container_metadata.rb

```diff
@@ -2,6 +2,7 @@ module Procedures::Repositories
   class IndexKatelloRepositoriesContainerMetatdata < ForemanMaintain::Procedure
     metadata do
       description 'Import container manifest metadata'
+      for_feature :repositories
```

Comment:
There is no such feature in Foreman Maintain!

Reply:
Huh, not sure how this slipped in there :)